core/arch/arm/plat-zynqmp/conf.mk

```diff
@@ -27,3 +27,23 @@ CFG_SHMEM_SIZE   ?= 0x10000000
 
 CFG_WITH_STATS ?= y
 CFG_CRYPTO_WITH_CE ?= y
+
+CFG_ZYNQMP_PM ?= $(CFG_ARM64_core)
+
+ifeq ($(CFG_RPMG_FS),y)
+$(call force,CFG_ZYNQMP_HUK,y,Mandated by CFG_RPMB_FS)
+endif
+
+ifeq ($(CFG_ZYNQMP_HUK),y)
+$(call force,CFG_ZYNQMP_AES,y,Mandated by CFG_ZYNQMP_HUK)
+$(call force,CFG_ZYNQMP_PUF,y,Mandated by CFG_ZYNQMP_HUK)
```

Comment:
Huk also depends on CFG_ZYNQMP_CSU_PUF because of zynqmp_csu_puf_reset.

Reply:
And on CFG_ZYNQMP_CSU_AES because of zynqmp_csu_aes_encrypt_data and zynqmp_csu_aes_decrypt_data.

Reply:
yes, the current configs are leftovers from incorrectly addressing previous renames...will fix!